webapp/product/models/product.py

```diff
@@ -15,11 +15,11 @@ class Meta:
     DEALED_STATUS = 3
 
     QUALITY_CHOICES = (
-        (1, '하'),
-        (2, '중하'),
-        (3, '중'),
-        (4, '중상'),
-        (5, '상'),
+        (1, 'very poor'),
+        (2, 'poor'),
+        (3, 'good'),
+        (4, 'very good'),
+        (5, 'excellent'),

```

Comment:
```suggestion
        (1, 'very_poor'),
        (2, 'poor'),
        (3, 'good'),
        (4, 'very_good'),
        (5, 'excellent'),

```

p1: 공백문자가 enum 값에 들어가는건 지양합니다

Reply:
수정했습니다.